peparser.cpp

```diff
@@ -1323,38 +1323,49 @@ namespace peparser
 		return false;
 	}
 
-	bool PEParser::DetectMIDLMarker(size_t diffStart, size_t diffSize, size_t& diffShift) const
+	bool PEParser::DetectMIDLMarker(size_t diffStart, size_t diffSize, size_t& diffShift, bool tlbCmpExpr) const
 	{
-		diffShift = 0;
+        diffShift = 0;
 
-		UsefulBlockMapRange markers = m_useful.equal_range(MidlTimestampSegment);
-		if(markers.first == markers.second) return false;
+        if (!tlbCmpExpr)
```

Comment:
If you want to hide it behind the switch, leave the old code when the switch is not set (not that it worked before with bug on line 1314, but still :))